src/utils/topics.js

```diff
@@ -23,12 +23,15 @@ export const topics = {
  * Object containing subtopic information from all topics
  */
 export function allSubtopics() {
-  return Object.entries(topics)
-    .flatMap(([, topicObj]) => Object.entries(topicObj.subtopics))
```

Comment:
@jkeat , @dmnisson,

I ran into a few issues with `flatMap`, one being that it wasn't supported in Electron and strangely enough, it may have been causing some of the CORS errors that we were experiencing running headless. I would love to know if you two are experiencing the same issues when running Cypress headless

Below are the steps to recreate it:
First running Cypress headless will give us some CORS related issues, we can fix that by [doing this](https://docs.cypress.io/guides/guides/web-security.html#Set-chromeWebSecurity-to-false-in-your). Run the test again and then I got an error saying that flatMap is not a function, hence the proposed refactor

What is strange to me is that the proposed refactor does not cause any CORS issues even though the cypress.json doesn't have the `chromeWebSecurity` property?